docs/guides/expectations/creating_custom_expectations/overview.md

```diff
@@ -1,18 +1,76 @@
 
-<Prerequisites>
+<Prerequisites></Prerequisites>
 
-- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
-OR
-- [Set up a working deployment of Great Expectations](../../../tutorials/getting_started/intro.md)
-    
-</Prerequisites>
+## Steps to create a custom Expectation
 
+A fully developed Expectation needs to do a lot of things:
+* Execute consistently across many types of data infrastructure
+* Render itself and its Validation Results into several different formats
+* Support Profiling against new data
+* Be maintainable, with good tests, documentation, linting, type hints, etc.
+
+In order to make development of Expectations as easy as possible, we've broken up the steps to create custom Expectations into a series of bite-sized steps. Each step can be completed in minutes. They can be completed (and contributed) incrementally, unlocking value at each step along the way.
+
+Grouped together, they constitute a Definition of Done for Expectations at each [Level of Maturity](/docs/contributing/contributing_maturity).
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+<i class="fas fa-circle" style={{color: "#dc3545"}}></i> An experimental Expectation...
+
+* Has a `library_metadata` object
+* Has a docstring, including a one-line short description
+* Has at least one positive and negative example case
+* Has core logic and passes tests on at least one `Execution Engine`
+
+<i class="fas fa-circle" style={{color: "#ffc107"}}></i> A beta Expectation...
+
+* Has basic input validation and type checking
+* Has all four statement `Renderers`: question, descriptive, prescriptive, diagnostic
+* Has default `Parameter Builders` and Domain hooks to support Profiling   
+* Has core logic that passes tests for all applicable `Execution Engines` and SQL dialects
+
+<i class="fas fa-check-circle" style={{color: "#28a745"}}></i> A production Expectation...
+
+* Passes all linting checks
+* Has all applicable Renderers, with fully typed and styled output
+* Has a full suite of tests, as determined by project code standards
+* Has passed a manual review by a code owner for code standards and style guides
+
+## How these docs are organized
+
+The docs in `Creating custom Expectations` focus on completing the four steps for experimental Expectations. Completing them will get to the point where your Expectation can be executed against one backend, with a couple tests to verify correctness, and a basic docstring and metadata to support diagonstics. Optionally, you can also publish experimental Expectations to the [Great Expectations open source gallery](https://greatexpectations.io/expectations) by following the steps [here](overview#publishing-your-expectation-as-an-open-source-contribution).
```

Comment:
- "couple **of** tests"
- "diagonstics" is misspelled.

Reply:
```suggestion
The docs in `Creating custom Expectations` focus on completing the four steps for experimental Expectations. Completing them will get to the point where your Expectation can be executed against one backend, with a couple of tests to verify correctness, and a basic docstring and metadata to support diagnostics. Optionally, you can also publish experimental Expectations to the [Great Expectations open source gallery](https://greatexpectations.io/expectations) by following the steps [here](overview#publishing-your-expectation-as-an-open-source-contribution).
```